### DISCUSSION TITLE
Browser server problems

### DISCUSSION THREAD

**Discussion by myang163** [foundation/foundation-emails#1091] [Q&A]
I got the email template from foundation working just fine but everytime I make adjustments I would always have to npm start on my terminal to refresh the page I'm working on, is there anyway to fix this so I don't have to always npm start?

**Comment by joeworkman** [comment]
I assume that you are using the Foundation CLI? The watch command has some issues at this time. We are working on a replacement CLI tool called Proton. I have not added support for Email into yet. But that should come soon.
